xtext/org.icyphy.linguafranca/src/org/icyphy/generator/GeneratorBase.xtend

```diff
@@ -993,14 +993,14 @@ abstract class GeneratorBase extends AbstractLinguaFrancaValidator {
     }
     
     /**
-     * Run the custom build command specified in "build" parameter
+     * Run the custom build command specified with the "build" parameter.
      */
     protected def runBuildCommand() {
-	var _buildCommand = newArrayList('-c', targetBuildCommand)
-        val buildCommand = createCommand("bash", _buildCommand)
-	
+        val cmd = this.targetBuildCommand.split("\\s+")
+        val buildCommand = createCommand(cmd.head, cmd.tail.toList)
```

Comment:
I think supporting multiple commands in one line would be a nice thing to have (`cmd1 && cmd2`). This fix doesn't seem to allow that since [createCommand](https://github.com/icyphy/lingua-franca/blob/0148dbecd1cbe6e10b89defcb0c1b9fa7a6139bf/xtext/org.icyphy.linguafranca/src/org/icyphy/generator/GeneratorBase.xtend#L1213) is expecting 1 command.

So I am thinking instead of `bash -c <string_cmd>`, how about using `sh -c <string_cmd>`? `sh` is more universal than `bash` and exists on all POSIX systems.

An alternative route is to have the user provide a list of commands and we can execute them one-by-one.